gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -531,7 +531,10 @@ public WritableByteChannel create(final StorageResourceId resourceId, CreateObje
               writeConditions,
               requesterPaysProjectId,
               BackOffFactory.DEFAULT);
+      channel.initialize();
+      return channel;
     } else {
+      BaseAbstractGoogleAsyncWriteChannel<?> channel;
```

Comment:
nit: channel can be directly initialized now.

Reply:
fixed.